src/atomic64/nightly.rs

```diff
@@ -59,31 +61,77 @@ impl Atomic<f64> for AtomicF64 {
             }
         }
     }
+
+    #[inline]
+    fn dec_by(&self, delta: Self::T) {
+        self.inc_by(-delta);
+    }
+}
+
+pub struct AtomicI64 {
+    inner: StdAtomicI64,
+}
+
+impl Atomic for AtomicI64 {
```

Comment:
For AtomicI64 and AtomicU64, I think all atomic operations can be `Ordering::Relaxed`

Reply:
let's keep it as original values in this PR. We can make another PR to discuss and optimize it.